localization/english/wc_mercenaries_l_english.yml

```
@@ -1,4 +1,130 @@
 ﻿l_english:
+# Ancient
+# Protector
+ mercenary_company_keepers_of_the_grove_article:0 "the "
+ mercenary_company_keepers_of_the_grove:0 "Keepers of the Grove"
+ mercenary_company_ironbarks_article:0 "the "
+ mercenary_company_ironbarks:0 "Ironbarks of [TITLE.GetNameNoTierNoTooltip]"
+# War
+ mercenary_company_stomping_roots_article:0 "the "
+ mercenary_company_stomping_roots:0 "Stomping Roots of [TITLE.GetNameNoTierNoTooltip]"
+# mercenary_company_ironbranch_guard_article:0 "the "
+ mercenary_company_ironbranch_guard:0 "Ironbranch's Guard"
+# Lore
+ mercenary_company_wisdom_forest_article:0 "the "
+ mercenary_company_wisdom_forest:0 "Wisdom of the Forest"
+ mercenary_company_elderleaves_article:0 "the "
+ mercenary_company_elderleaves:0 "Elderleaves of [TITLE.GetNameNoTierNoTooltip]"
+# Wind
+ mercenary_company_woodenbeaks_article:0 "the "
+ mercenary_company_woodenbeaks:0 "Woodenbeaks of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_rootclaws_article:0 "the "
+ mercenary_company_rootclaws:0 "Rootclaws of [TITLE.GetNameNoTierNoTooltip]"
+
+# Aqir
+# Qiraji
+# mercenary_company_cthun_guard_article:0 "the "
+ mercenary_company_cthun_guard:0 "C'thun's Guard"
+ mercenary_company_swarmslayers_article:0 "the "
+ mercenary_company_swarmslayers:0 "Swarmslayers of [TITLE.GetNameNoTierNoTooltip]"
+# Nerubian
+ mercenary_company_webspinners_article:0 "the "
+ mercenary_company_webspinners:0 "Webspinners of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_swarm_of_depth_article:0 "the "
+ mercenary_company_swarm_of_depth:0 "Swarm of the Depth"
+# Mantid
+ mercenary_company_blades_of_empress_article:0 "the "
+ mercenary_company_blades_of_empress:0 "Blades of the Empress"
+ mercenary_company_priests_of_yshaarj_article:0 "the "
+ mercenary_company_priests_of_yshaarj:0 "Priests of Y'Shaarj"
+
+# Arakkoa
+ mercenary_company_bladedancers_article:0 "the "
+ mercenary_company_bladedancers:0 "Bladedancers of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_sunguard_of_talon_king_article:0 "the "
+ mercenary_company_sunguard_of_talon_king:0 "Sunguard of the Talon King"
+
+# Centaur
+ mercenary_company_warriors_of_khan_article:0 "the "
+ mercenary_company_warriors_of_khan:0 "Warriors of the Khan"
+ mercenary_company_terror_of_steppe_article:0 "the "
+ mercenary_company_terror_of_steppe:0 "Terror of the Steppes"
+
+# Demonic
+# Eredar
+# mercenary_company_argus_demise_article:0 "the "
+ mercenary_company_argus_demise:0 "Argus' Demise"
+ mercenary_company_manari_lords_article:0 "the "
+ mercenary_company_manari_lords:0 "Man'ari lords of [TITLE.GetNameNoTierNoTooltip]"
```

Names are capitalized.
```suggestion
 mercenary_company_manari_lords:0 "Man'ari Lords of [TITLE.GetNameNoTierNoTooltip]"
```